docs/workflow/pygrb.rst

```diff
@@ -6,14 +6,35 @@ PyGRB: A GRB triggered CBC analysis workflow generator
 Introduction
```

Comment:
Can @sfairhur and @pannarale check they are happy with the documentation. I won't look over this myself.